libCacheSim-python/libcachesim/eviction.py

```diff
@@ -96,7 +101,7 @@ def process_trace(self, reader, start_req=0, max_req=-1) -> tuple[float, float]:
         byte_miss_ratio = 0.0
         if not isinstance(reader, Reader):
             # streaming generator
-            if isinstance(reader, (_ZipfRequestGenerator, _UniformRequestGenerator)):
+            if isinstance(reader, _GENERATOR_TYPES):
```

Comment:
In the ideal world, I hope we have streaming reader in C as well, but let's live with it